notebook_utils/berdl_notebook_utils/spark/database.py

```diff
@@ -96,32 +103,33 @@ def create_namespace_if_not_exists(
 
     if append_target:
         try:
-            namespace, db_location = generate_namespace_location(namespace, tenant_name)
-        except Exception as e:
-            print(f"Error creating namespace: {e}")
-            raise e
+            delta_namespace, db_location = generate_namespace_location(namespace, tenant_name)
+        except Exception:
+            # includes stack trace
+            logger.exception("Error creating namespace")
```

Comment:
If you use logger.exception within the context of an exception, it automatically includes the stack trace. Very generous!